dev_environment/e2e_test/setup_linux_dev_env.sh

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# this script needs to run as root account, check it
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root"
+    exit 1
+fi
+
+# Check for kernel version 5.15
+CURRENT_KERNEL_VERSION=$(uname -r | cut -d"." -f1-2)
+CURRENT_KERNEL_MAJOR_VERSION=$(echo "${CURRENT_KERNEL_VERSION}" | cut -d"." -f1)
+CURRENT_KERNEL_MINOR_VERSION=$(echo "${CURRENT_KERNEL_VERSION}" | cut -d"." -f2)
+ALLOWED_KERNEL_VERSION="5.15"
+ALLOWED_KERNEL_MAJOR_VERSION=$(echo ${ALLOWED_KERNEL_VERSION} | cut -d"." -f1)
+ALLOWED_KERNEL_MINOR_VERSION=$(echo ${ALLOWED_KERNEL_VERSION} | cut -d"." -f2)
+if [ "${CURRENT_KERNEL_MAJOR_VERSION}" -lt "${ALLOWED_KERNEL_MAJOR_VERSION}" ]; then
+  # If the current major version is less than the allowed major version, show an error message and exit.
+  echo "Error: Kernel ${CURRENT_KERNEL_VERSION} not supported, please update to ${ALLOWED_KERNEL_VERSION}."
+  exit
+fi
+if [ "${CURRENT_KERNEL_MAJOR_VERSION}" == "${ALLOWED_KERNEL_MAJOR_VERSION}" ]; then
+  # If the current major version is equal to the allowed major version, check the minor version.
+  if [ "${CURRENT_KERNEL_MINOR_VERSION}" -lt "${ALLOWED_KERNEL_MINOR_VERSION}" ]; then
+    # If the current minor version is less than the allowed minor version, show an error message and exit.
+    echo "Error: Kernel ${CURRENT_KERNEL_VERSION} not supported, please update to ${ALLOWED_KERNEL_VERSION}."
+    exit
+  fi
+fi
+
+# Get cpu architecture, arm or amd
+ARCH=$(uname -p)
+
+case $ARCH in
+  arm)
+    echo "Setting l3af dev environment for arm"
+    arch=arm64
+    ;;
+
+  aarch64)
+    echo "Setting l3af dev environment for arm"
+    arch=arm64
+    ;;
+
+  x86_64)
+    echo "Setting l3af dev environment for amd64"
+    arch=amd64
+    ;;
+
+  i386)
+    KERNEL=$(uname -m)
+    if [ "$KERNEL" = "x86_64" ];
+    then
+      echo "Setting l3af dev environment for amd64"
+      arch=amd64
+    elif [ "$KERNEL" = "i386" ];
+    then
+      echo "Setting l3af dev environment for i386"
+      arch=386
+    else
+      echo "The CPU kernel $KERNEL is not supported by the script"
+      exit 1
+    fi
+  ;;
+
+  *)
+    echo "The CPU architecture $ARCH is not supported by the script"
+    exit 1
+  ;;
+esac
+
+cd /root
+
+# install packages
+apt-get update 
+apt-get install -y software-properties-common wget
+
+# get grafana package
+wget -q -O - https://packages.grafana.com/gpg.key | apt-key add -
+echo "deb https://packages.grafana.com/oss/deb stable main" | tee -a /etc/apt/sources.list.d/grafana.list
+apt-get clean
+apt-get update
+# -end-
+
+# install all necessary packages
+# gcc-multilib      not existed for arm64 repos
```

Comment:
```suggestion
# gcc-multilib does not exist for arm64 repos
```

Reply:
In this line I am not making first letter capital, but other suggestions are done